backend/src/app/common/sample_contents.py

```diff
@@ -0,0 +1,236 @@
+    },
+    {
+        "id": "8yTxNtVr_d8",
+        "uploader": "TWICE",
+        "views": 0,
+        "title": "MORE & MORE",
+        "description": """Provided to YouTube by 'JYP Entertainment'
+
+        MORE & MORE · TWICE
+        
+        MORE & MORE
+        
+        Released on: 2020-06-01
+        
+        Auto-generated by YouTube.""",
+    },
+    {
+        "id": "0EK_M2taRIM",
+        "uploader": "IVE (아이브) - 주제",
+        "views": 0,
+        "title": "ELEVEN (ELEVEN)",
+        "description": """Provided to YouTube by Kakao Entertainment
+
+        ELEVEN (ELEVEN) · IVE
+        
+        ELEVEN
+        
+        ℗ 2021 STARSHIP Entertainment,under license to Kakao Entertainment
+        
+        Released on: 2021-12-01
+        
+        Author: 서지음
+        Author: Peter Rycroft
+        Author: Lauren Aquilina
+        Author: RYAN JHUN
+        Composer: Peter Rycroft
+        Composer: Lauren Aquilina
+        Composer: RYAN JHUN
+        Arranger: Lostboy
+        Arranger: RYAN JHUN
+        Arranger: Alawn
+        
+        Auto-generated by YouTube.""",
+    },
+    {
+        "id": "3laLavGghc0",
+        "uploader": "WHO IS sokodomo",
+        "views": 0,
+        "title": "MERRY-GO-ROUND (Feat. Zion.T & Wonstein) (Prod. by Slom)",
+        "description": """Provided to YouTube by Genie Music Corporation
+
+        MERRY-GO-ROUND (Feat. Zion.T & Wonstein) (Prod. by Slom) · sokodomo
+        
+        Show Me The Money 10 Episode 2
+        
+        ℗ 2021 Stone Music Entertainment
+        
+        Released on: 2021-11-13
+        
+        Auto-generated by YouTube.""",
+    },
+    {
+        "id": "YfQzz00Oc_M",
+        "uploader": "1theK (원더케이)",
+        "views": 0,
+        "title": "[MV] Loco(로꼬) _ It Takes Time(시간이 들겠지) (Feat. Colde)",
+        "description": """[MV] Loco(로꼬) _ It Takes Time(시간이 들겠지) (Feat. Colde)
+
+        [공지] 1theK YouTube는 MV를 유통하는 공식 채널로, 1theK에 업로드된 MV 조회수 또한 음악방송 순위에 반영됩니다.
+        
+        방송 및 각종 협업 프로젝트를 섭렵하고 있는 로꼬가 한 달 만에 신보를 들고 왔다. '시간이 들겠지'는 본인의 현재 심정을 솔직하게 잘 드러낸 곡이며 Colde가 피쳐링을 참여하여 로꼬의 감성을 한껏 끌어 올렸다. 로꼬는 다양한 음악 스타일과 개성이 뚜렷한 프로듀서 및 아티스트와의 협업으로 자신의 음악적 스펙트럼을 넓고 굳게 다져나가고 있다.
+        
+        ▶1theK FB  : http://www.facebook.com/1theK
+        ▶1theK TW : https://twitter.com/1theK
+        ▶1theK Kakao : https://goo.gl/otRpZc
+        
+        #Loco#로꼬#It_Takes_Time#시간이_들겠지#Colde#NEWRELEASE#MV#1theK#원더케이""",
+    },
+]
+
+sample_contents_query = """
+INSERT INTO contents(hashed_id, title, description, thumbnail_url, channel_id)
+VALUES"""
+for i, sample_content in enumerate(sample_contents):
+    sample_contents_query += f"(\"{sample_content['id']}\", "
+    sample_contents_query += f"\"{sample_content['title']}\", "
+    sample_contents_query += f"\"{sample_content['description']}\", "
+    sample_contents_query += '"fake_url", '
+    if i < len(sample_contents) - 1:
+        sample_contents_query += "1),\n"
+    else:
+        sample_contents_query += "1);"
```

Comment:
```suggestion
    sample_contents_query += f"\"{sample_content.get('thumbnail_url', 'fake_url')}\", "
    sample_contents_query += f"{sample_content.get('channel_id', 1)})\"
    
    endline =  ",\n" if  i < len(sample_contents) - 1 else ";"
    sample_contents_query += endline
```

값의 키와 default 값으로 적혀있으면 읽기 쉬울 것 같습니다

Reply:
좋아보입니다!